src/expressions/array-expr.md

```diff
@@ -86,3 +86,4 @@ The array index expression can be implemented for types other than arrays and sl
 [memory location]: ../expressions.md#place-expressions-and-value-expressions
 [path]: path-expr.md
 [slice]: ../types/slice.md
+[panic]: ../panic.md
```

Comment:
Place this in its sorted position, if you would, above `path`.

Reply:
I've sorted them, but wasn't sure how to handle instances where the tag starts with a backtick. I've treated the backtick as part of the text being sorted, i.e. they're lexicographically after `_` and before lower case letters.